balancer-labs/frontend-v2
# Hotfix/1.33.3

# Description

Due to a recent sorbet finance hack we needed to show an alert for a list of affected users. The snippet they provided us was very helpful but I thought we should have something that allows us to be more flexible with future exploits that we would want to alert our users of.

I've added an exploits watcher which reads from a list of active exploits from a constants file - so it'll alert the user with a CTA (in this case, a link to their tweet of the hack which has more detailed instructions)

Fixes # (issue)

## Type of change


## How should this be tested?

You can use the walletconnect spoofer to pretend to be another address (from the list of affected ones) connecting to the app, You should be able to see the alert.

## Checklist:
